MapboxNavigation/NavigationMapView.swift

```diff
@@ -1362,13 +1368,17 @@ extension NavigationMapView {
      - parameter extrudesBuildings: Switch which allows to highlight buildings in either 2D or 3D. Defaults to true.
      */
     public func highlightBuildings(at coordinates: [CLLocationCoordinate2D], in3D extrudesBuildings: Bool = true) {
-        highlightBuildings(with: Set(coordinates.compactMap({ buildingIdentifier(at: $0) })), in3D: extrudesBuildings)
+        buildingHighlightPollingTimer?.invalidate()
+        buildingHighlightPollingTimer = Timer.scheduledTimer(withTimeInterval: 0.5, repeats: true) { [weak self] timer in
```

Comment:
Every half-second seems somewhat aggressive, especially if the user ends up moving slowly (or stopped at a light) within that final third of a mile. Is there anything in `NavigationMapView.highlightBuildings(with:in3D:extrudeAll:)` that bails, avoiding style changes, if the call to that method is redundant?